extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -65,12 +65,32 @@ protected function getValueWithDefault( \SimpleXMLElement $xmlNode ) {
 		return $value;
 	}
 
+	protected function getRawValueWithDefault( \SimpleXMLElement $xmlNode ) {
+		$source = $this->getXmlAttribute( $xmlNode, self::DATA_SRC_ATTR_NAME );
+		$value = null;
+		if ( !empty( $source ) ) {
+			$value = $this->getInfoboxData( $source );
+		}
```

Comment:
Those few lines are duplicated in both getValueWithDefault and getRawValueWithDefault methods. Can we extract it?